app/Services/Manager.php

```diff
@@ -608,4 +608,24 @@ public function sendEmail($user, $contest, $params)
             return null;
         }
     }
+
+    /**
+     * Finds if user exists in a competition for a given contest
+     *
+     * @param app\Models\Contest         $contest
+     * @param string                     $id
+     * @return array|null
+     */
+    public function findUserInContest($contest, $id)
+    {
+        $userCompetitions = $contest->competitions->filter(function ($competition, $key) use ($id) {
+            $users = $competition->users->where('northstar_id', $id);
```

Comment:
I Agree! It should only ever be one ($users and $userCompetitions), but it (userCompetitions) does have the capability to hold more. I can make it more clear